chamber/models/__init__.py

```diff
@@ -30,17 +29,22 @@ class Options(object):
     def __init__(self, model):
         self.model = model
 
-        self.clean_before_save = True
-        self.clean_after_save = False
+        self.clean_pre_save = True
+        self.clean_post_save = False
 
-        self.clean_before_delete = False
-        self.clean_after_delete = False
+        self.clean_pre_delete = False
+        self.clean_post_delete = False
 
         self.atomic_save = False
         self.atomic_delete = False
 
         if hasattr(model, 'SmartMeta'):
-            self.clean_before_save = self._getattr('clean_before_save', self.clean_before_save)
+            self.clean_pre_save = self._getattr('clean_pre_save', self.clean_pre_save)
+            self.clean_post_save = self._getattr('clean_post_save', self.clean_post_save)
+            self.clean_pre_delete = self._getattr('clean_pre_delete', self.clean_pre_delete)
+            self.clean_post_delete = self._getattr('clean_post_delete', self.clean_post_delete)
+            self.atomic_save = self._getattr('atomic_save', self.atomic_save)
+            self.atomic_delete = self._getattr('atomic_delete', self.atomic_delete)
```

Comment:
atomicity is by default turned off and post clean too, because of performance (delete is not cleaned by default too)